mongo/tests/test_integration.py

```diff
@@ -514,3 +514,36 @@ def test_user_pass_options(check, instance_user, dd_run_check):
     with mock_pymongo("standalone"):
         # ensure we don't get a pymongo exception saying `Unknown option username`
         dd_run_check(check, extract_message=True)
+
+
+def test_db_names(check, instance_integration, aggregator, dd_run_check):
+    instance_integration['dbnames'] = ['admin', 'config', 'test', 'local', 'not_found']
```

Comment:
Nit: This test is asserting the same as test_standalone, I think it's missing the case of a db that exist but is not passed in the list. For that I guess you either need a new fixture or to be able to pass an exclude list of tags to _assert_metrics so it does not assert, for example, the ones with `db:local`